rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -523,7 +523,16 @@ public String eth_getTransactionCount(String address, String blockId) {
 
     public Block getBlockByJSonHash(String blockHash) {
         byte[] bhash = stringHexToByteArray(blockHash);
-        return this.blockchain.getBlockByHash(bhash);
+
+        try {
+            return this.blockchain.getBlockByHash(bhash);
+        } catch (Exception e) {
```

Comment:
as I understand this tries to catch the exception that may be thrown [here](https://github.com/rsksmart/rskj/blob/IRIS-3.1.0/rskj-core/src/main/java/co/rsk/crypto/Keccak256.java#L42). Can we simply validate `bhash` before calling `getBlockByHash` (the same way that it's being done in `Keccak256`) ?

Reply:
Addressed